css/style.css

```diff
@@ -1,6 +1,881 @@
+body {
+	margin: 0;
+	box-sizing: border-box;
+}
+
+.conteiner {
+	max-width: 1030px;
+	width: 100%;
+	margin: 0 auto;
+	box-sizing: border-box;
+
+}
+
+ a {
+ 	display: inline-block;
+ 	text-decoration: none;
+ }
+
+ .header {
+ 	width: 100%;
+ 	top: 0;
+ 	left: 0;
+ 	display: flex;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)